docs/docsite/rst/plugins/cache.rst

```diff
@@ -43,7 +50,7 @@ A custom cache plugin is enabled by dropping it into a ``cache_plugins`` directo
 Enabling Inventory Cache Plugins
 --------------------------------
 
-Inventory may be cached using a file-based cache plugin (like jsonfile). Check the specific inventory plugin to see if it supports caching.
```

Comment:
do we have a ticket to enable using cache plugins from collections?

Reply:
Yes, there is this https://github.com/ansible/ansible/pull/56469.